packages/core/test/system.test.ts

```diff
@@ -127,4 +127,39 @@ describe('system', () => {
       `"Actor with system ID 'test' already exists."`
     );
   });
+
+  it('should be accessible in inline custom actions', () => {
+    const machine = createMachine({
+      invoke: {
+        src: createMachine({}),
+        systemId: 'test'
+      },
+      entry: (_, __, { system }) => {
+        expect(system!.get('test')).toBeDefined();
+      }
+    });
+
+    interpret(machine).start();
+  });
+
+  it('should be accessible in referenced custom actions', () => {
```

Comment:
I only fixed this for "custom actions" aka functions. Previously it was only provided to inline ones but references ones didn't have that privilege. 

It's still not accessible to `assign`, `sendTo` (!), and more. Do we want to add it to every builtin action type?

Reply:
I'd say so 👍

Reply:
Added here: 4adcba4f648042f249269eb3441c88ccb4836136